src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,80 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final int MAXIMAL_CAPACITY = 1 << 30;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size = 0;
+    private ArrayList<Entry<K,V>>[]table = new ArrayList[DEFAULT_INITIAL_CAPACITY];
```

Comment:
When creating any type of list (ArrayList or LinkedList) create an object of type List (more generic, interface) with reference to ArrayList class. It is then type safe list. It also adheres to:
Open/Closed Principle: Declaring variables and parameters using interfaces adheres to the Open/Closed Principle, which states that software entities should be open for extension but closed for modification.
Dependency Inversion Principle: High-level modules should not depend on low-level modules; both should depend on abstractions. By using interfaces, you adhere to this principle.
e.g. 
`List<String> al = new ArrayList<String>();`

Reply:
ok, I hope I understood it correctly, and now this declaration is good.